### ISSUE TITLE
Test sourceset doesn't have minecraft in its classpath

### ISSUE THREAD

**Issue by filloax** [CrackedPolishedBlackstoneBricksMC/minivan#3]
Hi, for my small library I tried switching to minivan from vanillagradle, but my test module (needed to avoid headache in a codec-related API I made) doesn't seem to have Minecraft in its classpath after switching. Is this normal?

**Comment by filloax** [comment]
A workaround was using the manual version and adding it to testImplementation, like this:

```kts
val minecraft = minivan.minecraftBuilder()
    .version(minecraftVersion)
    .accessWideners("src/main/resources/${modid}.accesswidener")
    .build()
    .minecraft;

// Fix minivan not adding minecraft to test classpath
project.dependencies.add("testImplementation", project.files(minecraft.minecraft))
minecraft.dependencies.forEach { project.dependencies.add("testImplementation", it) }
```
(kotlin DSL)

**Comment by quat1024** [comment]
Yeah I just slap the dependency into `compileOnly` by default: https://github.com/CrackedPolishedBlackstoneBricksMC/minivan/blob/c5db5684cac44a713afa185874ef0e6e1bf4760b/src/main/java/agency/highlysuspect/minivan/MinivanExt.java#L59-L64

Never noticed since I don't really write a lot of tests 😅 I should add it to testCompileOnly or smtn; this should probably be made configurable as well
